src/components/gallery/upload/Dropzone.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+  import { getImagesFromWNFS, uploadImageToWNFS } from '$lib/gallery'
+
+  /**
+   * Detect when a user drags a file in or out of the dropzone to change the styles
+   */
+  let isDragging = false
+  const handleDragEnter: () => void = () => (isDragging = true)
+  const handleDragLeave: () => void = () => (isDragging = false)
+
+  /**
+   * Process files being dropped in the drop zone and ensure they are images
+   * @param event
+   */
+  const handleDrop: (event: DragEvent) => Promise<void> = async event => {
+    // Prevent default behavior (Prevent file from being opened)
+    event.preventDefault()
+
+    const files = Array.from(event.dataTransfer.items)
+
+    console.log(`${files.length} file${files.length > 1 ? 's' : ''} dropped`)
+
+    // Iterate over the dropped files and upload them to WNFS
+    await Promise.all(
+      files.map(async (item, index) => {
+        if (item.kind === 'file') {
+          const file: File = item.getAsFile()
+
+          // If the dropped files aren't images, we don't want them!
+          if (!file.type.match('image/*')) {
+            console.error('Please upload images only')
```

Comment:
What would you think about showing this error in a notification?

Reply:
Yep, absolutely! Thanks for catching that 👍🏼